docs/source/development/commands_reference.md

```diff
@@ -323,22 +323,25 @@ kedro run
+| `kedro run --from-nodes=<node_name1>,<node_name2>`                  | A list of node names which should be used as a starting point                                                                                                                                                                                           |
+| `kedro run --to-nodes=<node_name1>,<node_name1>`                    | A list of node names which should be used as an end point                                                                                                                                                                                               |
+| [DEPRECATED] `kedro run --node=<node_name1>,<node_name2>`           | Run only nodes with specified names. <br /> Multiple instances allowed. <br /> NOTE: This flag will be deprecated in `Kedro 0.19.0`. Use the following flag `--nodes` instead.                                                                          |
+| `kedro run --nodes=<node_name1>,<node_name2>`                       | Run only nodes with specified names.                                                                                                                                                                                                                    |
+| `kedro run --runner=<runner_name>`                                  | Run the pipeline with a specific runner                                                                                                                                                                                                                 |
+| `kedro run --async`                                                 | Load and save node inputs and outputs asynchronously with threads                                                                                                                                                                                       |
+| `kedro run --env=<env_name>`                                        | Run the pipeline in the env_name environment. Defaults to local if not provided                                                                                                                                                                         |
+| [DEPRECATED] `kedro run --tag=<tag_name1>,<tag_name2>`              | Run only nodes which have any of these tags attached. <br /> Multiple instances allowed.                                                                                                                                                                |
+| `kedro run --tags=<tag_name1>,<tag_name2>`                          | Run only nodes which have any of these tags attached. <br /> NOTE: This flag will be deprecated in `Kedro 0.19.0`. Use the following flag `--tags` instead.                                                                                             |
```

Comment:
I think the note is on the wrong line here. It should be in the for above.

Reply:
Yes, I put it in the wrong place. Made the change in dd12b68e535d000816d0cfff523b60bd7eb1064f